java/ql/lib/semmle/code/java/security/PathSanitizer.qll

```diff
@@ -64,7 +64,11 @@ private predicate exactPathMatchGuard(Guard g, Expr e, boolean branch) {
   )
 }
 
-private class ExactPathMatchSanitizer extends PathInjectionSanitizer {
+/**
+ * A sanitizer that protects against path injection vulnerabilities
+ * by checking for a matching path.
+ */
+class ExactPathMatchSanitizer extends PathInjectionSanitizer {
```

Comment:
It's a bit weird that this is the only non-private sanitizer exposed by this library, but I understand why you did it. No need to change anything else, just taking mental note that it may make sense to expose all these sanitizers for consistency.